editoast/editoast_authz/src/lib.rs

```diff
@@ -47,6 +47,8 @@ pub async fn ensure_latest_authorization_model(
 pub enum Error<StorageError: std::error::Error> {
     #[error("unknown subject {0}")]
     UnknownSubject(i64),
+    #[error("unknown user {0}")]
+    UnknownUser(String),
```

Comment:
```suggestion
    #[error("unknown user {identity}")]
    UnknownUser { identity: String },
```

nit, but more explicit